src/CommunityToolkit.Maui/Behaviors/Validators/ValidationBehavior.shared.cs

```diff
@@ -421,8 +421,15 @@ void UpdateStyle(in VisualElement view, bool isValid)
 
 	void ResetValidationTokenSource(CancellationTokenSource? newTokenSource)
 	{
-		validationTokenSource?.Cancel();
-		validationTokenSource = newTokenSource;
+		try
+		{
+			validationTokenSource?.Cancel();
```

Comment:
I believe we should dispose the tokenSource as well

```suggestion
			validationTokenSource?.Cancel();
			validationTokenSource?.Dispose();
```

Reply:
@pictos I made this change.